vuepress/docs/next/tutorials/consume/external-db-ms.md

```diff
@@ -0,0 +1,203 @@
+2. For each microservice that is using an external DB, add an `env` section with the Spring username, password, URL, and driver as required for the connection.
+
+The following `entando.json` shows the specs required for a sample microservice named YOUR-MS-SQL: 
+
+```json
+{
+  "microservices": [
+        {
+            "name": "YOUR-MS-SQL",
+            "dbms": "none",
+            "env": [
+                {
+                    "name":"SPRING_DATASOURCE_USERNAME",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "ed1d60fc-285a480f-external-books-service-mysql-secret",
+                            "key": "SPRING_DATASOURCE_USERNAME"
+                        }
+                    }
+                },
+                {
+                    "name":"SPRING_DATASOURCE_PASSWORD",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "ed1d60fc-285a480f-external-books-service-mysql-secret",
+                            "key": "SPRING_DATASOURCE_PASSWORD"
+                        }
+                    }
+                },
+                {
+                    "name":"SPRING_DATASOURCE_URL",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "ed1d60fc-285a480f-external-books-service-mysql-secret",
+                            "key": "SPRING_DATASOURCE_URL"
+                        }
+                    }
+                },
+                {
+                    "name":"SPRING_JPA_DATABASE_PLATFORM",
+                    "value": "org.hibernate.dialect.MySQL8Dialect"
+                }
+            ]
+        }
+    ]
+    ...
+    "name": "YOUR-BUNDLE-NAME",
+    "version": "1.0.0",
+    "description": "a description",
+    "type": "bundle"
+}
+```
+3. Pack and publish your bundle:
+```
+  ent bundle pack
+  ent bundle publish
+```
+
+### Create and Apply a Plugin Configuration Secret
+Prior to installing a bundle, a plugin configuration Secret must be created to provide a different datasource for each tenant. Create a Secret for each microservice in the bundle to provide a series of values required by the Spring Dynamic Datasource Routing for every tenant as needed.  
+
+1. Retrieve the bundle ID and the plugin code:
+ ```
+ ent ecr get-bundle-id docker://registry.hub.docker.com/YOUR-ORG/YOUR-BUNDLE
+ ```
+ ```
+ ent ecr get-plugin-code YOUR-ORG/YOUR-MS-NAME --repo=docker://registry.hub.docker.com/YOUR-ORG/YOUR-BUNDLE
+ ```
+ 
+2. Create the JSON plugin configuration Secret for the microservice plugin with a name that reflects the bundle ID and plugin code as Kuternetes is not tenant aware.
+
+   The name should begin with "pn" plus the concatenation of the bundle ID, plugin code, and microservice name, connected by dashes. The name can be appended with suffixes such as "SECRET" for better identification.  
```

Comment:
The name should begin with the plugin code and must be connected to the suffix `-conf`
The plugin configuration secret (differently from other referenced secrets) does not admin any other string in the middle